frontend/src/component/admin/auth/AuthSettings.tsx

```diff
@@ -35,17 +34,14 @@ export const AuthSettings = () => {
             label: 'Google',
             component: <GoogleAuth />,
         },
+        {
+            label: 'SCIM',
+            component: <ScimSettings />,
+        },
     ].filter(
         (item) => uiConfig.flags?.googleAuthEnabled || item.label !== 'Google',
     );
 
-    if (isEnterprise()) {
-        tabs.push({
-            label: 'SCIM',
-            component: <ScimSettings />,
-        });
-    }
```

Comment:
I need some guidance here.

Isn't SSO an Enterprise-only feature anyways? 

AFAIK there's no way to get `authenticationType === 'enterprise'` in a hosted Pro instance. A Pro instance would simply display "Your Unleash instance is managed by the Unleash team."

This PR simplifies our logic to rely on the plan instead of the auth type and protect SSO behind the Enterprise plan.

If we're okay with this, then maybe we should protect the route from showing up at all.